CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ SET(BUILD_SERVER ON CACHE BOOL
     "Build the Srch2Server executable. Defaults to ON")
 SET(BUILD_RELEASE ON CACHE BOOL "Build Release. Default ON")
 SET(BUILD_TEST ON CACHE BOOL "Build tests. Default ON")
-SET(BUILD_UTILITY OFF CACHE BOOL "Build utilities. Default OFF")
+SET(BUILD_UTILITY ON CACHE BOOL "Build utilities. Default ON")
```

Comment:
Comment by @JavierJia : we turn the utility build "ON" in order to build a Chinese dictionary that is needed by the analyzer test.